data/hai/hai reveal 4 middle.txt

```diff
@@ -1728,11 +1778,116 @@ mission "Hai Reveal [A12] Into the Lion's Den"
+			branch "Danforth Alpha Hunt"
+				"Danforth in mission sequence" == 2
+
+			# Danforth is not in any mission sequence.
+			action
+				# Danforth mission sequence 1 is Hai Reveal 4-6
+				"Danforth in mission sequence" = 1
+				log `Alerted Turner and Danforth to the situation. Been sent back to <planet> to ensure the safety of the diplomats.`
 			`	"I'll be there with a fleet soon. In the meantime, you need to go back to <planet> to ensure the safety of the diplomatic corps. Get them out if need be, otherwise just be there with a ship so you can monitor the situation."`
 				accept
 
+			# Danforth is at Farpoint during the Alpha Hunt sequence.
+			label "Danforth Alpha Hunt"
+			action
+				log `Alerted Turner and Danforth to the Hai situation. Was urged to finish finding and eliminating the Alpha enclave as soon as possible, so the Navy can deal with this new threat.`
+			`	"I can't deal with this problem personally until we finish hunting the Alphas. The Hai - even the Unfettered - are capable of restraint; they're not amoral and power-hungry like the Alphas. We need to solve the Alpha problem decisively, before they can do any real harm. After that, I'll arrange formal discussions with the Hai."`
+				decline
+
+			# Danforth is at Farpoint during the Sestor invasion, and he is NOT waiting for you to drop the bomb.
+			label "Danforth Alpha Sestor Invasion"
+			action
+				log `Alerted Turner and Danforth to the Hai situation. Was urged to finish finding and eliminating the Alpha enclave as soon as possible, so the Navy can deal with this new threat.`
+			`	From back on your ship it is easy enough to use the station's network to place a call. You get in contact with Danforth to brief him on what's going on, with the possibility of an Unfettered fleet coming through the wormhole in a matter of days. He seems exasperated. "We have few ships to spare, but I'll send enough to make our presence felt at the wormhole. In the meantime, I suggest you focus on the current invasion of human space before trying to spark any new ones."`
+				decline
+
+			# Danforth is at Farpoint, under attack, waiting for you to drop the bomb.
+			label "Danforth Alpha Sestor Invasion: drop the bomb"
+			action
+				log `Alerted Turner and Danforth to the Hai situation. Was urged to finish finding and eliminating the Alpha enclave as soon as possible, so the Navy can deal with this new threat.`
+			`	From back on your ship it is easy enough to use the station's network to place a call. You get in contact with Danforth to brief him on what's going on, with the possibility of an Unfettered fleet coming through the wormhole in a matter of days. You can barely hear his response over the explosions around his base. "What the hell are you doing there? Haven't you noticed the Navy is occupied right now? Drop the bomb and then we'll worry about the Hai."`
```

Comment:
Danforth doesn't swear with words elsewhere, so maybe change it to be a bit less harsh.
```suggestion
			`	From back on your ship it is easy enough to use the station's network to place a call. You get in contact with Danforth to brief him on what's going on, with the possibility of an Unfettered fleet coming through the wormhole in a matter of days. You can barely hear his response over the explosions around his base. "What on Farpoint are you doing there? Haven't you noticed the Navy is occupied right now? Drop the bomb and then we'll worry about the Hai."`
```
(I know, you could also use Earth, but...)